tests/versions/apm_server.yml

```diff
@@ -1,7 +1,7 @@
 APM_SERVER:
-  - '6.x'
+  - '6.7;--release'
```

Comment:
6.7 is not released yet, you will need to use the github branch (by defining `6.7`)